apps/cowswap-frontend/src/common/pure/ClickableAddress/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { MouseEvent } from 'react'
+
+import { getChainInfo } from '@cowprotocol/common-const'
+import { shortenAddress } from '@cowprotocol/common-utils'
+import { HoverTooltip } from '@cowprotocol/ui'
+
+import CopyHelper, { CopyIcon } from 'legacy/components/Copy/CopyMod'
+
+import * as styledEl from './styled'
+
+const getTarget = (address: string, chainId: number) => {
```

Comment:
Nitpick, we decided to simplify the way we work with chainId in the app.
We always assume that chainId is `SupportedChainId`, so no need to check it every time.
We do the checks in the top level using `useIsProviderNetworkUnsupported()` so we don't need to check it on every single case.